src/get-started/install/_get-sdk-win.md

```diff
@@ -39,7 +39,7 @@ In the Flutter Console, run the following command to see if there are any
 dependencies you need to install to complete the setup:
 
 ```terminal
-$ flutter doctor
+$ flutter doctor [-v]
```

Comment:
The prose before this line contains "run the following command", but by adding `[-v]` this line becomes more a "usage synopsis". Options that come to mind:

- Revert this change
- Drop the brackets so that it becomes `$ flutter doctor -v`
- Keep the change but:
  - Update the prose (to not say "run the following command"), _and_
  - Remove `terminal` as the code-block language 
  - Remove the leading `$`

Reply:
This point wasn't addressed.